services/management/dbaas/version_service_client.go

```diff
@@ -158,6 +180,42 @@ func (c *VersionServiceClient) Matrix(ctx context.Context, params componentsPara
 	return &vsResponse, nil
 }
 
+func (c *VersionServiceClient) RecommendedComponentVersion(ctx context.Context, product, component string) (string, *componentVersion, error) {
+	m, err := c.Matrix(ctx, componentsParams{
+		product: product,
```

Comment:
we need to pass operator version here to get recommended DB version for exact operator version